packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -314,13 +339,9 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
           </ComboBoxInputContainer>
           <ComboBoxDropdown {...dropdownMenuProps}>
             <ComboBoxMenu>
-              {filteredOptions.length > 0 ? (
-                filteredOptions.map(getMenuItem)
-              ) : (
-                <MenuExtraContent>
-                  <Text isMuted>{emptyStateMessage}</Text>
-                </MenuExtraContent>
-              )}
+              {filteredOptions.length > 0
+                ? filteredOptions.map(getMenuItem)
```

Comment:
Won't this end up generating react elements for all options no matter whether you have virtualisation enabled or not?

Reply:
I added `enableVirtualization` check, so it only passes empty content component